### DISCUSSION TITLE
Does Keycloak 19.0.0-legacy use/have the new Admin UI?

### DISCUSSION THREAD

**Discussion by ErikHumphrey** [keycloak/keycloak#13649] [Q&A] [answered]
Not sure if Legacy uses the new React-based admin UI rather than the AngularJS admin UI, especially now that it's enabled by default.

**Comment by ssilvert** [comment]
The old UI is still available for at least a few more releases.  Just go to Realm settings --> Themes and select "keycloak" for the admin theme.

**Reply by ErikHumphrey** [reply]
Yup, being removed in Keycloak 21. Just checking that the legacy version also has the new theme, but I guess it makes sense that it would since the KC16->17 Quarkus change wouldn't prevent something like that!

Thanks!

**Comment by ErikHumphrey** [comment, marked as answer]
Yes.
